index.html

```diff
@@ -0,0 +1,201 @@
+                                <li>350gm chicken</li>
+                                <li>1/2 teaspoon ginger paste</li>
+                                <li>1/2 teaspoon red chilli powder</li>
+                                <li>1/4 teaspoon cumin powder</li>
+                                <li>salt as required</li>
+                                <li>2 teaspoon virgin olive oil</li>
+                                <li>1/2 cup hung curd</li>
+                                <li>1/2 teaspoon garlic paste</li>
+                                <li>1/4 teaspoon corriander powder</li>
+                                <li>1/4 teaspoon powdered black pepper</li>
+                                <li>1 teaspoon garam masala powder</li>
+                            </ul>
+                            <h4>Instructions/Method</h4>
+                            <p>
+                                <h5>Step 1: Wash the chicken pieces</h5>
+                                To prepare this easy chicken recipe, wash the chicken pieces and prick them with a fork.
+                                Now, in a large bowl, combine hung curd, ginger paste, garlic paste, olive oil, coriander powder red chilli powder, cumin powder, black pepper, salt and garam masala.<br>
+                                Stir all the ingredients together until well blended.
+                                <h5>Step 2: Grill it to perfection and serve hot!</h5>
+                                    <p>
+                                        Now add the chicken pieces and apply the masala mixture all over them.
+                                        Cover and refrigerate for at least 8 hours or up to 24 hours.
+                                        Once done, preheat the oven at 190 degree Celsius.
+                                        Remove the chicken pieces from the marinade and place them on a grilling tray.
+                                        Grill them for 1 hour, flipping them after 30 minutes so that both sides can cook evenly.
+                                        Check if the chicken has fully cooked by taking one piece out and cutting it with a knife.
+                                        Serve hot with salad or a dip.
+
+                                    </p>
+                            </p>
+                        </span>
+                    </p>
+                    <div class="buttons">
+                        <button onclick="myFunction()" id="myButton">View More</button>
+                        <!-- <i class="fa-light fa-heart fa-4x"></i> -->
+                        <!-- <span id="myButton"><i class="fa-regular fa-heart"></i></span> -->
+                        <a href="https://kit.fontawesome.com/a462069d14.js" crossorigin="anonymous"><i class="fa-regular fa-heart"></i></a>
+                    </div>
+            </div>
+            <div>
+                <img src="pexels-valeria-boltneva-1639562.jpg" alt="tantalizing burger">
```

Comment:
its good practice to have images in its own folder because assumming your project will have thousands of images zikikaa njee zitakua mob sana na ngumu kumanage

create a new folder assets then ndani folder ingine images

add all your images apo kwa hiyo folder and link them penye unazitumia as  src="./assets/images/pexels-valeria-boltneva-1639562.jpg"